extension/src/repository/model/collect.ts

```diff
@@ -34,12 +40,13 @@ const transformToAbsTree = (
 
 export const collectTree = (
   dvcRoot: string,
-  absLeafs: Set<string>,
-  trackedRelPaths = new Set<string>()
```

Comment:
[F] We can do this because we no longer need the `exp show` data to work out which files we can pull directly.